lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -408,4 +408,27 @@ defmodule IEx.AutocompleteTest do
     assert expand('take(') == {:yes, '', ['take(enumerable, amount)']}
     assert expand('derive(') == {:yes, '', ['derive(protocol, module, options \\\\ [])']}
   end
+
+  @tag :tmp_dir
+  test "path completion inside strings", %{tmp_dir: dir} do
```

Comment:
We need a bunch more tests given the complexity of the code. :) I would also create all the files upfront and then have an extensive assertion suite.

Reply:
Added more assertions, please let me know if you think we need more.